packages/react/src/patterns/sub-patterns/TableOfContents/README.md

```diff
@@ -81,13 +86,15 @@ Add the following line in your `.env` file at the root of your project.
 
 ## Props
 
-| Name           | Required | Data Type | Default Value | Description                                                                      |
-| -------------- | -------- | --------- | ------------- | -------------------------------------------------------------------------------- |
-| `children`     | NO       | Array     | null          | Content to display next to the side nav.                                         |
-| `menuItems`    | NO       | Array     | null          | Array of menu item objects to render within the side nav. See `menuItems` below. |
-| `menuLabel`    | NO       | String    | `Jump to`     | Placeholder value for menu label                                                 |
-| `theme`        | NO       | String    | `white`       | Defines the color theme for the pattern (See `Theme` bellow)                     |
-| `stickyOffset` | NO       | Number    | null          | Defines the offset for the sticky column                                         |
+| Name             | Required | Data Type | Default Value | Description                                                                      |
+| ---------------- | -------- | --------- | ------------- | -------------------------------------------------------------------------------- |
+| `children`       | NO       | Array     | null          | Content to display next to the side nav.                                         |
+| `headingContent` | NO       | Component | null          | Content to be displayed above the navigation menu.                               |
+| `menuItems`      | NO       | Array     | null          | Array of menu item objects to render within the side nav. See `menuItems` below. |
+| `menuLabel`      | NO       | String    | `Jump to`     | Placeholder value for menu label                                                 |
+| `theme`          | NO       | String    | `white`       | Defines the color theme for the pattern (See `Theme` bellow)                     |
+| `stickyOffset`   | NO       | Number    | null          | Defines the offset for the sticky column                                         |
+| `menurule`       | NO       | Boolean   | null          | Defines if the menu ruler will be rendered                                       |
```

Comment:
Super minor, it looks like it should be `menuRule`

Reply:
@jeffchew I've corrected it, thank you!